cmd/antrea-agent/agent.go

```diff
@@ -136,6 +137,8 @@ func run(o *Options) error {
 	enableBridgingMode := enableAntreaIPAM && o.config.EnableBridgingMode
 	enableNodePortLocal := features.DefaultFeatureGate.Enabled(features.NodePortLocal) && o.config.NodePortLocal.Enable
 	enableMulticluster := features.DefaultFeatureGate.Enabled(features.Multicluster) && o.config.Multicluster.Enable
+	// L7NetworkPolicy is not supported on Windows yet.
+	l7NetworkPolicyEnabled := features.DefaultFeatureGate.Enabled(features.L7NetworkPolicy) && !runtime.IsWindowsPlatform()
```

Comment:
is this necessary? I thought that adding `L7NetworkPolicy` to the `unsupportedFeaturesOnWindows` map would guarantee an early error?

Reply:
removed